### DISCUSSION TITLE
Clinical practice involving notes

### DISCUSSION THREAD

**Discussion by KevinKloiber** [MIT-LCP/mimic-code#1667] [MIMIC-IV] [answered]
Hello,

I am engaged in an academic research project that has led me to delve into the MIMIC IV dataset, with a particular focus on the clinical notes.

My inquiry leans more toward understanding the procedural aspects of the note composition rather than the data itself. I am uncertain if this forum is the appropriate venue for such a question, but I was wondering how much freedom physicians have when writing the notes and whether the notes are exclusively penned by physicians, or are a collaborative effort involving assistants or other medical personnel. I am also especially interested in the discharge summary and how many "degrees of freedom" authors have when writing those or, framed differently, how much variation is there between the physicians writing those notes.  I would be grateful for any insights the community can offer on this matter. 

Thank you.

**Comment by alistairewj** [comment, marked as answer]
When I have seen physicians create discharge summaries they follow a template but have freedom otherwise and we're editing plain text. They would copy paste from relevant other sections of the EHR (e. g. pertinent lab results). @lceli would be able to answer better if he has time.

**Reply by KevinKloiber** [reply]
Thanks a lot for your reply! Really appreciate it. That helps a lot for my research question since I was wondering whether the discharge section could be used as a sort of quality indicator of physicians to pass down information to patients and explain relevant information in simple terms.
